js/guides-4.8.js

```diff
@@ -0,0 +1,351 @@
+// Polyfill Element.matches()
```

Comment:
guides.js is never used. This is used for diffing purposes only, since we're using a kinda dumb versioning scheme to allow for better browser caching.